lib/Onyx.js

```diff
@@ -794,6 +802,36 @@ function notifyCollectionSubscribersOnNextTick(key, value) {
     Promise.resolve().then(() => keysChanged(key, value));
 }
 
+/**
+ * This method waits for prior modifications to complete before running the next modification.
+ *
+ * @param {Function} nextModification
+ * @param {String} key If "key" is not set then the modification will be queued after all prior modifications completed.
+ * @returns {Promise}
+ */
+function waitPendingWrites(nextModification, key) {
```

Comment:
Maybe default to `null` if we are going to have a check for it? Otherwise, I would not use `null` and would just check to see if we have a key at all.

Reply:
doing `value == null` instead of `value === null` actually means, that we check for undefined too. So we are already checking if the key is defined or not

Reply:
better to be strict/explicit here